plugins/Files/css/files.css

```diff
@@ -121,8 +138,10 @@ hr {
 .file.selected {
 	background-color: #ddd;
 }
+.file:hover {
+	background-color: #ECECEC;
+}
 .file.anchor {
-	border: 1px solid #00CBA0;
```

Comment:
Why get rid of the anchor selection box? I thought it was neat and made the effect of shift+click selections explicit